tutorials/cosmwasm.md

```diff
@@ -29,8 +30,8 @@ this tutorial.
 
 For this tutorial, we will need `go` and `jq` installed on your machine. You can install them by running our script:
 
-```bash
-curl -sSL https://rollkit.dev/install-go.sh | bash -s go1.22.3
+```bash-vue
+curl -sSL https://rollkit.dev/install-go.sh | bash -s {{constants.golangVersion}}
```

Comment:
This seems to be getting rendered incorrectly at least in https://rollkit.dev/guides/connect-local-da, please make sure that's not the case.

Reply:
agh,yes